datafusion/core/src/datasource/memory.rs

```diff
@@ -19,9 +19,9 @@
 
 use datafusion_physical_plan::metrics::MetricsSet;
 use futures::StreamExt;
-use hashbrown::HashMap;
 use log::debug;
 use std::any::Any;
+use std::collections::HashMap;
```

Comment:
I introduced `hashbrown::HashMap` in https://github.com/apache/arrow-datafusion/pull/8283, now I am replacing it with `std::collections::HashMap`.